templates/chttpd.ini.j2

```diff
@@ -23,3 +23,6 @@ socket_options = {{ couchdb_socket_options }}
 ; For more socket options, consult Erlang's module 'inet' man page.
 ;socket_options = [{recbuf, 262144}, {sndbuf, 262144}, {nodelay, true}]
 {% endif %}
+
+[couch_httpd_auth]
+secret = {{ couchdb_httpd_auth_secret }}
```

Comment:
Will be created by the DB if it doesn't exist and added to `ssl.ini` (as the last config file).